### ISSUE TITLE
Images not loading in react JS 

### ISSUE THREAD

**Issue by ak3311g** [liabru/matter-js#1303]
When i am importing image for texture in it's showing image not found even after giving right path

**Comment by dewang002** [comment]
can you provide SS for more clarity

**Comment by ggorlen** [comment]
A runnable minimal example of the code that reproduces the issue would be ideal. Are you serving the images somewhere, or just pointing to a local file?

**Comment by ak3311g** [comment]
I am pointing it to local I tried to place it in same directory as the component file and also tried with public folder but in both condition its not working

and one more thing I am facing i don't know why but the mouse and mouseconstraints also not working on my react app

**Comment by ak3311g** [comment]
![image](https://github.com/liabru/matter-js/assets/49281815/9fdbe66d-9c38-4b8f-a766-4f05458eb01f)

this is the issue I am currently facing after creating a new project because I have deleted my previous project out of frustration :)

**Comment by ak3311g** [comment]
for some time it started working but again same issue

**Comment by dewang002** [comment]
Do you contribute on daily basis?
I am thinking to participate in hackathon would you like to join

**Comment by ak3311g** [comment]
@dewang002 yeah I am ready and also this issue is fixed I think or I figured out another way to do this I need to first place items in public folder and run the engine and render just after defining it i am creating boxes and borders first then doing the running thats why I saw this method on internet but when I run the engine and render before adding boxes and border it runs fine.
